test/unit/qualityTest.test.js

```diff
@@ -174,30 +174,48 @@ describe('Model quality test', () => {
     initFishData();
   });
 
-  // TODO: (maddie) fix this test to work with new fishData.colors setup + re-enable
-  // test('Color test', async () => {
-  //   const trainSize = TRAIN_SIZE;
-  //
-  //   const idsByColor = {
-  //     red: [2, 5],
-  //     green: [3, 4],
-  //     blue: [0, 6]
-  //   };
-  //
-  //   for (const [color, ids] of Object.entries(idsByColor)) {
-  //     console.log(`${color}`);
-  //     const labelFn = (fish) => ids.includes(fish.colorPalette.index) ? ClassType.Like : ClassType.Dislike;
-  //     const result = await performTrials({
-  //       numTrials: NUM_TRIALS,
-  //       trainSize: trainSize,
-  //       testSize: 100,
-  //       labelFn: labelFn
-  //     });
-  //     analyzeConfusionMatrix(trainSize, result);
-  //     expect(result.precision).toBeGreaterThanOrEqual(0.85);
-  //     expect(result.recall).toBeGreaterThanOrEqual(0.5);
-  //   }
-  // });
+  test('Color test', async () => {
+    const trainSize = TRAIN_SIZE;
+
+    const idsByColor = {
+      red: [6, 7],
+      green: [2, 3],
+      blue: [4, 5]
+    };
+
+    const colorMatch = (rgb, colorIds) => {
+      const rgbs = Object.values(fishData.colors).map(color => color.rgb);
+
+      let match = false;
+      colorIds.forEach(id => {
```

Comment:
I find the inner `return` statement from this block confusing, can we use the `for(const id of colorIds) {}` syntax instead so it just sets the `match` variable normally and then `break`s? Feel free to push back if you don't like that 😁

Reply:
ah, yeah, that would read better. i'll update!